### DISCUSSION TITLE
mhtml web page serving

### DISCUSSION THREAD

**Discussion by timttran** [Badgerati/Pode#1085] [Ideas]
Hi, 

Any chance for a Pode to serve mhtml web pages in the Public folder?
This would be an awesome feature!

**Comment by Badgerati** [comment]
HI,

What happens when you try to view an mhtml page currently in the browser, served via Pode?

Reading here: https://en.wikipedia.org/wiki/MHTML it "should" work, as the parsing of the file is done by the browser rather than the server - unless the wrong MIME type is being sent 🤔

**Comment by timttran** [comment]
Hi,

I did try it.
It prompts to download the file with the SaveAs dialog popup; whereas other html files work as expected.
I tested it by saving the html files to the Public\Test folder.

Please see if you can get this to work...would be awesome!

Thanks,
Tim
________________________________

**Comment by Badgerati** [comment]
Hi,

In which case it might be the MIME type Pode is returning on the response. I'll create an issue from this to test what happens if the IME type is changed :)

**Comment by timttran** [comment]
Great! Can't wait to test it out.

Thanks Matthew.


-Tim

________________________________
